UnitTests/BraintreePayPalTests/BTPayPalLineItem_Tests.swift

```diff
@@ -8,41 +8,41 @@ class BTPayPalLineItem_Tests: XCTestCase {
        
         lineItem.upcType = .UPC_A 
-        let requestParams = lineItem.requestParameters();
-        XCTAssertEqual(requestParams["upc_type"] as! String, "UPC-A");
-       
+        var requestParams = lineItem.requestParameters()
+        XCTAssertEqual(requestParams["upc_type"], "UPC-A");
+
         lineItem.upcType = .UPC_B 
-        requestParams = lineItem.requestParameters();
-        XCTAssertEqual(requestParams["upc_type"] as! String, "UPC-B");
-       
+        requestParams = lineItem.requestParameters()
+        XCTAssertEqual(requestParams["upc_type"], "UPC-B")
+
         lineItem.upcType = .UPC_C 
-        requestParams = lineItem.requestParameters();
-        XCTAssertEqual(requestParams["upc_type"] as! String, "UPC-C");
-       
+        requestParams = lineItem.requestParameters()
+        XCTAssertEqual(requestParams["upc_type"], "UPC-C")
+
         lineItem.upcType = .UPC_D 
-        requestParams = lineItem.requestParameters();
-        XCTAssertEqual(requestParams["upc_type"] as! String, "UPC-D");
-       
+        requestParams = lineItem.requestParameters()
+        XCTAssertEqual(requestParams["upc_type"], "UPC-D")
+
         lineItem.upcType = .UPC_E 
-        requestParams = lineItem.requestParameters();
-        XCTAssertEqual(requestParams["upc_type"] as! String, "UPC-E");
-       
-        lineItem.upcType = .UPC_1 
-        requestParams = lineItem.requestParameters();
-        XCTAssertEqual(requestParams["upc_type"] as! String, "UPC-1");
-       
+        requestParams = lineItem.requestParameters()
+        XCTAssertEqual(requestParams["upc_type"], "UPC-E")
+
         lineItem.upcType = .UPC_2 
-        requestParams = lineItem.requestParameters();
-        XCTAssertEqual(requestParams["upc_type"] as! String, "UPC-2");
+        requestParams = lineItem.requestParameters()
+        XCTAssertEqual(requestParams["upc_type"]!, "UPC-2")
+
+        lineItem.upcType = .UPC_5
+        requestParams = lineItem.requestParameters()
+        XCTAssertEqual(requestParams["upc_type"], "UPC-5")
     }
 
     func testKindStringReturnsCorrectValue() {
-        let lineItem = BTPayPalLineItem(quantity: "1", unitAmount: "10", name: "item-name", kind: .debit)
-        let requestParams = lineItem.requestParameters();
-        XCTAssertEqual(requestParams["kind"] as! String, "debit");
-        
+        var lineItem = BTPayPalLineItem(quantity: "1", unitAmount: "10", name: "item-name", kind: .debit)
+        var requestParams = lineItem.requestParameters()
+        XCTAssertEqual(requestParams["kind"], "debit");
+
         lineItem = BTPayPalLineItem(quantity: "1", unitAmount: "10", name: "item-name", kind: .credit)
-        requestParams = lineItem.requestParameters();
-        XCTAssertEqual(requestParams["kind"] as! String, "credit");
+        requestParams = lineItem.requestParameters()
+        XCTAssertEqual(requestParams["kind"], "credit");
     }
```

Comment:
this one?

Reply:
See https://github.com/braintree/braintree_ios/pull/1176#discussion_r1457962075